Prefabs/Locomotion/DestinationLocations/SharedResources/Scripts/DestinationLocationFacade.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace VRTK.Prefabs.Locomotion.DestinationLocations
+{
+    using UnityEngine;
+    using UnityEngine.Events;
+    using Malimbe.MemberChangeMethod;
+    using Malimbe.MemberClearanceMethod;
+    using Malimbe.XmlDocumentationAttribute;
+    using Malimbe.PropertySerializationAttribute;
+    using Zinnia.Rule;
+    using Zinnia.Data.Attribute;
+
+    /// <summary>
+    /// The public interface into the DestinationLocation Prefab.
+    /// </summary>
+    public class DestinationLocationFacade : MonoBehaviour
+    {
+        #region Location Settings
+        /// <summary>
+        /// Determines if the location is in the locked and unusable state.
+        /// </summary>
+        [Serialized]
+        [field: Header("Location Settings"), DocumentedByXml]
+        public bool IsLocked { get; set; }
+        /// <summary>
+        /// Apply the rotation of the custom destination location to the selected action output.
+        /// </summary>
+        [Serialized]
+        [field: DocumentedByXml]
+        public bool ApplyDestinationRotation { get; set; } = true;
```

Comment:
`AppliesDestinationRotation`

Reply:
We use `Apply` everywhere else (e.g. ApplyOffset, ApplyPositionOffsetOnAxis)

I'd say we raise a ticket to change it globally in the future (as a global change will break existing prefab setups) instead of creating an inconsistency now